resources/v1/projects/requests.go

```diff
@@ -14,11 +14,12 @@ type ListOptsBuilder interface {
 
 // ListOpts contains parameters for filtering a List request.
 type ListOpts struct {
-	Cluster  string `h:"X-Limes-Cluster-Id"`
-	Detail   bool   `q:"detail"`
-	Area     string `q:"area"`
-	Service  string `q:"service"`
-	Resource string `q:"resource"`
+	Cluster  string   `h:"X-Limes-Cluster-Id"`
+	Detail   bool     `q:"detail"`
+	Area     []string `q:"area"`
+	Service  []string `q:"service"`
+	Resource []string `q:"resource"`
+	Rates    string   `q:"rates"`
```

Comment:
Since only a limited set of values are allowed, this would be better expressed as an enum type like:

```
type RatesDisplay string

const (
  WithoutRates RatesDisplay = ""
  WithRates RatesDisplay = "true"
  OnlyRates RatesDisplay = "only"
)
```

Also, please check that setting this field to `""` correctly omits the parameter from the query string. Otherwise we have to find another way, since omitting `?rates` will be what most users want.

(And again, the same concern applies below for `GetOpts`.)

Reply:
I tested all the enums with `--debug` flag of limesctl. The queries get built as expected.